lib/driver.ts

```diff
@@ -413,6 +413,7 @@ class AndroidDriver
   activateApp = activateApp;
   queryAppState = queryAppState;
   isAppInstalled = isAppInstalled;
+  mobileBackgroundApp = background;
```

Comment:
`mobileBackgroundApp` and `background` definitions are not the same. `mobileBackgroundApp` sets the background argument to -1 by default, which changes the default behavior

Reply:
That's right (default one)

Reply:
added as a new mobile command: https://github.com/appium/appium-android-driver/pull/988/commits/2a847cfdba60b0a05a2d6270d3ed07269bc45df6